arguments.go

```diff
@@ -0,0 +1,66 @@
+package cmds
```

Comment:
Need to be able to continue treating the argument reader as an `io.ReadCloser` so we can't use `bufio.Scanner`.